components/Header.js

```diff
@@ -0,0 +1,41 @@
+import { View, Text, StyleSheet } from 'react-native';
+
+const Header = () => {
+    return (
+        <View style={styles.headerContainer}>
+            <Text style={styles.title}>Habitual</Text>
+        </View>
+    )
+}
+
+export default Header;
+
+const styles = StyleSheet.create({
+    headerContainer: {
+        marginBottom: 50,
+        backgroundColor: '#3A376F',
+        height: 80,
+        width: 300, 
+        alignItems: 'center',
+        justifyContent: 'center',
+        // shadowColor: "#869684",
```

Comment:
- Great opportunity to work on the drop shadow for ios and android. Seems like this is more subtle on mobile devices than web. 
- Header looks exact to wireframe!